### PR TITLE
Feat/solidity v0.8

### PR DESCRIPTION
**Pull request by paulrberg**

### CODE REVIEW

File: contracts/FyToken.sol [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /* SPDX-License-Identifier: LGPL-3.0-or-later */
-pragma solidity ^0.7.0;
+pragma solidity ^0.8.0;
 
 import "@paulrberg/contracts/access/Admin.sol";
-import "@paulrberg/contracts/math/CarefulMath.sol";
+import "@paulrberg/contracts/math/Exponential.sol";
```
Comment (scorpion9979) [inline contracts/FyToken.sol:5]: Where's Exponential being used in this fuke?

Comment (paulrberg) [reply contracts/FyToken.sol:5]: Good catch.

**Review by scorpion9979 — APPROVED** [review]
Looks good to merge overall except for that one line I commented on.

**Review by surbhiaudichya — APPROVED** [review]
Looks good.

**Comment by paulrberg** [comment]
Thanks!

**Pull request merged**